### ISSUE TITLE
Request timeout issue when ping_interval=None and ping_timeout=None are set.

### ISSUE THREAD

**Issue by Mynk96** [mobilityhouse/ocpp#626]
I dont get 'CALL' response in time its gets timeout error, but soon as I disconnect the charger from the server and when it reconnects on its own the 'CALL' are working fine .
Its happening with some of the charger they send heartbeat at regular interval but don't respond to Call like(remoteStart,clearCache,reset etc.)
It mostly happens after charger is idle for long time , it doesn't respond to any CALL but regularly send heartbeat ,until it reconnects

FYI
This particular vendor don't support ping/pong message so have diasabled them in in web socket. hope that is not causing any issue

 websockets.serve(handler, "0.0.0.0", 9000 subprotocols=["ocpp1.6"], ping_interval=None, ping_timeout=None)

**Comment by Yoshida-Taisei** [comment]
@Mynk96 

I’ve faced the same issue.

Here’s what I observed:

* WebSocket communication isn’t interrupted. Ping messages continue to be sent, and no errors are occurring.
* However, the charging station (CS) doesn’t respond to requests, and the calls are timing out.

One solution I tried was reducing the `ping interval to less than 90 seconds`. When I set it to 180 seconds, the problem reappeared, so using a shorter interval might help.

```python

websockets.connect(
    cs_path, ping_interval=90, subprotocols=["ocpp1.6"]
) as ws:

```

On my Central server, the issue doesn’t happen with other OCPP-compatible chargers, so it’s likely that the problem lies with the charger itself.
